src/code_review.gleam

```diff
@@ -17,8 +13,8 @@ import gleam/result
 
 // RUNNING THE LINTER ----------------------------------------------------------
 
-pub fn main() -> Nil {
-  case run(on: project.root()) {
+pub fn main(rules: List(Rule)) -> Nil {
```

Comment:
`main` taking args is surprising to me. I think the expectation is someone imports and runs this module instead of running it from the cli directly but if that's the case I think it would be better to rename this to `run` (and rename run below to `do_run`) to avoid any confusion.

Reply:
I agree. Let's just not have a main, refactor comin up